### PR TITLE
Adding K8S

### PR DESCRIPTION
**Pull request by subnet-dev**

### CODE REVIEW

**Comment by subnet-dev** [comment]
Hello, do you know when my pool can be added ? Thank you !

File: registry.json [hunk 1]
```diff
@@ -3382,5 +3382,20 @@
     "telegram": "yadavnie",
     "operatorName": "Yadav/Nadeem",
     "github": "adamoonpool"
+  },
+  {
+    "ticker": "K8S",
+    "poolId": "0960deeaabd7ca77a970e2932d06ec594c902301973041c6d9b1dc2e",
+    "social": {
+      "facebook": "",
+      "twitter": "",
```
Comment (Bez625) [inline registry.json:3391]: Please can you add some social media presence? We ask that you have some way of reaching delegators via social

**Review by Bez625 — CHANGES_REQUESTED** [review]

File: registry.json (hunk 1 above)
Comment (subnet-dev) [reply registry.json]: Twitter account added ! ✅

**Review by Bez625 — APPROVED** [review]

**Review by Bez625 — APPROVED** [review]

**Comment by subnet-dev** [comment]
Awesome, thanks!

**Pull request merged**